native_client/dotnet/DeepSpeechClient/DeepSpeech.cs

```diff
@@ -87,40 +87,40 @@ public unsafe int GetModelSampleRate()
         /// Evaluate the result code and will raise an exception if necessary.
         /// </summary>
         /// <param name="resultCode">Native result code.</param>
-        private void EvaluateResultCode(ErrorCodes resultCode)
+        private string EvaluateResultCode(ErrorCodes resultCode)
         {
             switch (resultCode)
```

Comment:
There is no need anymore for the switch here, you could just `return DS_ErrorCodeToErrorMessage(resultCode).IntPtrToString();`

Reply:
Ok I will remove the switch.

Reply:
IMHO you can remove the whole method

Reply:
Okay I will remove it.